lib/class-wp-block.php

```diff
@@ -169,19 +169,30 @@ public function __get( $name ) {
 	/**
 	 * Generates the render output for the block.
 	 *
+	 * @param array $options         Optional options object.
+	 *        bool  $options.dynamic Optionally set to false to avoid using the block's render_callback.
+	 *
 	 * @return string Rendered block output.
 	 */
-	public function render() {
+	public function render( $options = array() ) {
```

Comment:
Why is this update needed?

Reply:
>Still, I did have to add a couple of things to the server rendering process: A way to skip inner blocks when rendering a block so that the Query Loop block doesn't recurse infinitely when it has a child Post Content block, and a way to skip dynamic rendering so that the Query Loop block can call its render function without recursing.